autopsy.sls

```diff
@@ -1,12 +1,26 @@
 autopsy:
+  '4.17.0':
+    full_name: 'Autopsy'
+    {% if grains['cpuarch'] == 'AMD64' %}
+    installer: 'https://github.com/sleuthkit/autopsy/releases/download/autopsy-4.17.0/autopsy-4.17.0-64bit.msi'
+    uninstaller: 'https://github.com/sleuthkit/autopsy/releases/download/autopsy-4.17.0/autopsy-4.17.0-64bit.msi'
+    {% else %}
+    installer: 'https://github.com/sleuthkit/autopsy/releases/download/autopsy-4.17.0/autopsy-4.17.0-32bit.msi'
+    uninstaller: 'https://github.com/sleuthkit/autopsy/releases/download/autopsy-4.17.0/autopsy-4.17.0-32bit.msi'
+    {% endif %}
+    install_flags: '/qn /norestart'
+    uninstall_flags: '/qn /norestart'
+    msiexec: True
+    locale: en_US
+    reboot: False
   '4.3.0':
     full_name: 'Autopsy'
     {% if grains['cpuarch'] == 'AMD64' %}
-    installer: 'https://downloads.sourceforge.net/project/autopsy/autopsy/4.3.0/autopsy-4.3.0-64bit.msi'
-    uninstaller: 'https://downloads.sourceforge.net/project/autopsy/autopsy/4.3.0/autopsy-4.3.0-64bit.msi'
+    installer: 'https://netcologne.dl.sourceforge.net/project/autopsy/autopsy/4.3.0/autopsy-4.3.0-64bit.msi'
+    uninstaller: 'https://netcologne.dl.sourceforge.net/project/autopsy/autopsy/4.3.0/autopsy-4.3.0-64bit.msi'
     {% else %}
-    installer: 'https://downloads.sourceforge.net/project/autopsy/autopsy/4.3.0/autopsy-4.3.0-32bit.msi'
-    uninstaller: 'https://downloads.sourceforge.net/project/autopsy/autopsy/4.3.0/autopsy-4.3.0-32bit.msi'
+    installer: 'https://netcologne.dl.sourceforge.net/project/autopsy/autopsy/4.3.0/autopsy-4.3.0-32bit.msi'
+    uninstaller: 'https://netcologne.dl.sourceforge.net/project/autopsy/autopsy/4.3.0/autopsy-4.3.0-32bit.msi'
```

Comment:
Please continue to use `downloads.sourceforge.net...`. These other subdomains are dynamic and may not always exist.

Reply:
Good morning,

your wish is my order 😃 

I don't know how salt then downloads these packages because with curl, wget or http request I always get redirected to the sourceforge page.
Therefore, I had entered the mirror here. - But I just noticed that it redirects and aborts via wget.